Sprintfinity3902/Entities/Link/Player.cs

```diff
@@ -20,7 +20,7 @@ public class Player : AbstractEntity, IPlayer
         private static int THIRTEEN = 13;
         private static int ONE_HUNDRED_TWENTY = 120;
         private static int ONE_HUNDRED_NINETY_THREE = 193;
-        private static int INITIAL_HEALTH = 6;
+        private static int INITIAL_HEALTH = 10;
```

Comment:
Eventually change this back, but fine for testing.

Reply:
If we want to keep it at 6 for Master I am totally cool with that! I should have brought it up in the meeting. Kyle could keep it a 6 for his pr and that would be fine!